src/util/gitHub.ts

```diff
@@ -99,11 +99,32 @@ export function createCommitComment(token: string, rr: GitHubRepoRef, comment: C
     return axios.post(url, comment, authHeaders(token));
 }
 
-function authHeaders(token: string) {
+export function createRepo(token: string, rr: GitHubRepoRef, description: string, priv: boolean): AxiosPromise {
+    const config = authHeaders(token);
+    return axios.get(`${rr.apiBase}/orgs/${rr.owner}`, config)
+        .then(result => {
+            // We now know the owner is an org
+            return `${rr.apiBase}/orgs/${rr.owner}/repos`;
+        }, err => {
+            // We now know the owner is an user
```

Comment:
I still think this is bad -- there are other errors besides this one. However, this is not part of the PR, it's just been there a while.

Reply:
why is it bad?

Reply:
"the owner is a user" is not the only reason the call can fail